packages/params/src/index.ts

```diff
@@ -186,6 +194,17 @@ export const FINALIZED_ROOT_GINDEX = 105;
  */
 export const FINALIZED_ROOT_DEPTH = 6;
 export const FINALIZED_ROOT_INDEX = 41;
+
+// TODO: add correct values here
+export const EXECUTION_PAYLOAD_GINDEX = 105;
```

Comment:
TODO: cal and update corrected

Reply:
looks to be 33

```ts
types.ssz.capella.LightClientUpdate.getPathInfo(['attestedHeader', 'execution']).gindex
// 33
```

Reply:
the proof is from the block body to payload, so updated  accordingly